[DOC] Introduce concerns example in Getting Started [ci-skip] (#44684)
rails/rails: guides/source/getting_started.md

app/models/concerns
```

> In the example below, we will create a new feature for our blog that does not use concerns, but would benefit from them. Then, we will do a refactor where we begin using concerns in order to make our code more maintainable and reusable.

Reviewer: I toyed with the wording a bit to make it more concise.  Does the following still address the point of confusion sufficiently?

```suggestion
In the example below, we will implement a new feature for our blog that would benefit from using a concern. Then, we will create a concern, and refactor the code to use it, making the code more DRY and maintainable.
```

(Note that "DRY" has already been introduced [at the start of the guide](https://guides.rubyonrails.org/v7.0.0/getting_started.html#what-is-rails-questionmark).)

Author: Yes it does!
